### DISCUSSION TITLE
My breadcrumbs are stuck

### DISCUSSION THREAD

**Discussion by Sirfara** [twbs/bootstrap#39412] [Q&A]
Hi everyone! 

I'm trying to solve an issue with my website, and maybe you can help me. I am developing a very basic website (it only has four pages, and three of them refer to the "Home"), and I want to add breadcrumbs.

I decided to put them at the bottom of the website, since they will only refer to one page. The thing is that with my current code they are fixed in the same possition, so they will remain on place while scrolling. As a result, they overlap the footer of the site (like "floating" over it). This only happens when the breadcrumbs are coded with Bootstrap. Here is my code:

```html
html
<!--breadcrumbs-->
<nav aria-label="breadcrumbs">
<div class="breadcrumbs"><ol class="breadcrumb">
    <li class="breadcrumb-item"><a href="home.html">Home</a></li>
    <li class="breadcrumb-item active" aria-current="page">Project</li>
  </ol>
</div>
</nav>
``` 

I am not sure if I will keep this breadcrumbs, but i would like to know how to solve this issue for future problems. 
Any help will be really welcome :).

**Comment by julien-deramond** [comment]
Hi @Sirfara 
Would you have more context to share such as the complete layout, or the rest of the page, maybe the CSS linked to `.breadcrumbs` which is specific to your website? The code you're sharing is only using the `.breadcrumb` and `.breadcrumb-item` classes from Bootstrap which wouldn't cause this issue.

**Reply by Sirfara** [reply]
Hi @julien-deramond 

Thank you for your answer. I think it was a problem with the cookies. Once I restarted (instead of refreshing) the browser, the issue was gone. I only started coding recently, so I will assume it was magic U-_-.

Thank you anyway!

**Comment by DillonAdriyan** [comment]
It seems like the issue you're facing is related to the positioning of the breadcrumbs. To ensure they don't overlap with the footer when scrolling, you can adjust their positioning using CSS. Here's an example of how you might solve this issue:

```html
<!-- HTML -->
<nav aria-label="breadcrumbs" class="fixed-bottom">
  <div class="breadcrumbs">
    <ol class="breadcrumb">
      <li class="breadcrumb-item"><a href="home.html">Home</a></li>
      <li class="breadcrumb-item active" aria-current="page">Project</li>
    </ol>
  </div>
</nav>
```

And in your CSS file:

```css
/* CSS */
.breadcrumbs {
  position: fixed;
  bottom: 0;
  width: 100%;
  background-color: white; /* Adjust background color if needed */
  z-index: 999; /* Set a high z-index to ensure breadcrumbs appear above other elements */
}
```

This CSS code will fix the breadcrumbs to the bottom of the page, preventing them from overlapping with the footer while scrolling. Adjust the `background-color` and other styles as needed to match your website's design.

This solution assumes that your footer is not fixed at the bottom of the viewport. If your footer is also fixed, additional adjustments might be necessary.

from chatGpt

**Reply by Sirfara** [reply]
Thank you!
